html-tables/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Tables</title>
+  </head>
+  <body>
+    <h1>Student Grade Table</h1>
+    <table>
+      <thead>
+        <tr>
+          <th><strong>Student Name</strong></th>
+          <th><strong>Course</strong></th>
+          <th><strong>Grade</strong></th>
```

Comment:
`th` will be bold by default & will be read as a heading by the screenreader, so no need to include `strong`